src/mcnp/mcnp6/CMakeLists.txt

```diff
@@ -109,9 +160,12 @@ elseif (MCNP_VERSION STREQUAL "610")
 elseif (MCNP_VERSION STREQUAL "611")
   add_definitions(-DTHREAD="MCNP")
   add_definitions(-DTVERS="6.1.1b")
+elseif (MCNP_VERSION STREQUAL "620")
+  add_definitions(-DTHREAD="MCNP_6.20")
+  add_definitions(-DTVERS="6.2.0")
 endif ()
 add_definitions(-DLODDAT="${ENV_DATE}")
-if (MCNP_VERSION STREQUAL "611")
+if (MCNP_VERSION STREQUAL "611" OR MCNP_VERSION STREQUAL "620")
```

Comment:
If comparisons like these are going to be true with some kind of permanence can we use a CMake `GREATER "610"` comparison here so this extends well to multiple versions of MCNP6? 
(Ideally it would be `GREATER_EQUALS` but that isn't available until for CMake < 3.7 I guess)

Reply:
I think it is safer to refer to specific versions....

Reply:
Meaning these definitions will need to change often?

Reply:
I wouldn't really consider MCNP's release schedule to be "often"...

Reply:
Yes, but only as often as new MCNP versions are released - every 2 hrs or so?